index.bs

```diff
@@ -47,7 +47,7 @@ url: https://drafts.csswg.org/css-transitions-1/#transitions; type: dfn; text: C
     }
 </style>
 
-<h2 id="basic-principles">Principles behind design of Web APIs</h2>
+<h2 id="basic-principles">Principles Behind Design of Web Features</h2>
```

Comment:
We should add a section defining what we mean by API and how it differentiates from a Feature.